src/test/java/seedu/duke/command/SummaryCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.Item;
+import seedu.duke.SpendingList;
+import seedu.duke.Ui;
+import seedu.duke.exceptions.InvalidMonthException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class SummaryCommandTest {
+    public static ArrayList<Item> initList(Item... items) {
+        return new ArrayList<>(Arrays.asList(items));
+    }
+
+    public static SpendingList initSpendingList(Item... items) {
+        return new SpendingList(initList(items));
+    }
+
+    SpendingList spendingList = initSpendingList(
+            new Item("noodle", "S$", 1.5),
+            new Item("fish", "S$", 10),
+            new Item("books", "S$", 8.9)
+    );
+
+    @Test
+    public void execute() {
```

Comment:
Since validSummaryMonth, summaryYear and summary throw no exception, you can move them outside the try block.

Reply:
since the **xxx.execute()** method throws an exception, **summaryYear.excecute()** should be in the try block to catch exception if there is any